packages/engine/src/domain/commitments/__tests__/index.ts

```diff
@@ -183,6 +224,50 @@ export function threeWayLedgerCommitment(params: ThreeWayLedgerCommitmentParams)
       return unreachable(idx);
   }
 }
+export function ledgerState(params: LedgerCommitmentParams): SignedState {
+  const turnNum = params.turnNum;
+  const isFinal = params.isFinal || false;
+  const balances = params.balances || twoThree;
+  let proposedBalances = params.proposedBalances || blankBalance;
+  let furtherVotesRequired = 0;
+  if (params.proposedBalances) {
+    furtherVotesRequired = 1;
+    proposedBalances = params.proposedBalances;
+  }
+  const outcome = [
+    {
+      assetHolderAddress: ETH_ASSET_HOLDER_ADDRESS,
+      allocation: balances.map(b => {
+        return {destination: convertAddressToBytes32(b.address), amount: b.wei};
+      })
+    }
+  ];
+  const proposedOutcome: Outcome = [
+    {
+      assetHolderAddress: ETH_ASSET_HOLDER_ADDRESS,
+      allocation: proposedBalances.map(b => {
+        return {destination: convertAddressToBytes32(b.address), amount: b.wei};
+      })
+    }
+  ];
```

Comment:
This pattern seems to come up a lot. It might be nice to have a helper function e.g.
```ts
const proposedOutcome = ethOutcome(balances)
```